src/rust/engine/src/nodes.rs

```diff
@@ -413,22 +448,109 @@ impl ExecuteProcess {
   }
 }
 
+///
+/// A Node that represents a set of processes to execute on specific platforms.
+///
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+pub struct MultiPlatformExecuteProcess(MultiPlatformExecuteProcessRequest);
+
+impl MultiPlatformExecuteProcess {
+  fn get_platform_variant(variant_candidate: String) -> Platform {
+    match variant_candidate.as_ref() {
+      "darwin" => Platform::Darwin,
+      "linux" => Platform::Linux,
+      "none" => Platform::None,
+      _ => unreachable!(),
```

Comment:
It's unclear why this is the case, so it would be good to reference the python-side enum, and for the python side enum to reference this (with comments, until we have a better way to do that).

Reply:
Can you elaborate more on what is unclear. Is it the unreachable statement or the seemingly arbitrary strings?

Reply:
This code asserts that it is impossible for this string to contain anything other than the other values. Why is that?: A comment should explain.